### PR TITLE
privacy-resources: sort information, fix cloudflare-tor link & move it to information

### PR DESCRIPTION
**Pull request by Mikaela**
Pointed out at https://codeberg.org/crimeflare/cloudflare-tor/issues/28#issuecomment-5327

Preview: https://deploy-preview-1205--privacytools-io.netlify.com/#resources

### CODE REVIEW

File: _includes/sections/privacy-resources.html [hunk 1]
```diff
@@ -35,7 +35,7 @@ <h3>Tools</h3>
     by the late Aaron Swartz and is currently managed by Freedom of the Press Foundation.</li>
   <li><a href="https://pack.resetthenet.org/"><strong>Reset The Net - Privacy Pack</strong></a> - Help fight to end mass surveillance. Get these tools to protect yourself and your friends.</li>
   <li><a href="https://secfirst.org/"><strong>Security First</strong></a> - Umbrella is an Android app that provides all the advice needed to operate safely in a hostile environment.</li>
-  <li><a href="https://notabug.org/themusicgod1/cloudflare-tor"><strong>Block Cloudflare MiTM Attack</strong></a> - Firefox add-on to detect and block Cloudflare MITM attack.</li>
+  <li><a href="https://codeberg.org/crimeflare/cloudflare-tor"><strong>Block Cloudflare MiTM Attack</strong></a> - Firefox add-on to detect and block Cloudflare MITM attack.</li>
```
Comment (nitrohorse) [inline _includes/sections/privacy-resources.html]: While we're here :smile: Maybe the title of this link needs to be updated to suggest the "repository"? Or we update the link to point to the actual Firefox extension code? https://codeberg.org/crimeflare/cloudflare-tor/src/branch/master/addon_firefox

But then that makes wonder why not use the direct AMO store link? https://addons.mozilla.org/en-US/firefox/addon/bcma/ But then that's not even in English...

Comment (Mikaela) [reply _includes/sections/privacy-resources.html]: > I just have a problem with the description

>> Firefox add-on to detect and block Cloudflare MITM attack.

> as there is so much of other documentation here and that description downplays its significance and a Chrome user might not even click it, but I have no idea what I should say. Suggestions welcome :)

* me at https://codeberg.org/crimeflare/cloudflare-tor/issues/28#issuecomment-5668

Comment (nitrohorse) [reply _includes/sections/privacy-resources.html]: What's also weird is that the AMO store version is a year old. Very confusing.

Comment (nitrohorse) [reply _includes/sections/privacy-resources.html]: Hmm... looking at https://codeberg.org/crimeflare/cloudflare-tor/src/branch/master/addon_firefox, it contains two Firefox extensions. Both readme's have broken download links. Should we just remove this link until it's easily consumable for users?

Comment (Mikaela) [reply _includes/sections/privacy-resources.html]: I moved it to information which I hope is a better place for it.

File: _includes/sections/privacy-resources.html [hunk 2]
```diff
@@ -14,14 +14,15 @@ <h3>Guides</h3>
...
-  <li><a href="https://keybase.io/"><strong>Keybase.io</strong></a> - Get a public key, safely, starting just with someone's social media username.</li>
-  <li><a href="https://freedom.press/"><strong>Freedom of the Press Foundation</strong></a> - Supporting and defending journalism dedicated to transparency and accountability since 2012.</li>
-  <li><a href="https://www.erfahrungen.com/mit/anonymisierung/t/"><strong>Erfahrungen.com</strong></a> - German review aggregator website of privacy-related services.</li>
   <li><a href="https://tosdr.org/"><strong>Terms of Service; Didn't Read</strong></a> - "I have read and agree to the Terms" is the biggest lie on the web. We aim to fix that.</li>
-  <li><a href="https://privacy.net/us-government-surveillance-spying-data"><strong>privacy.net</strong></a> - What does the US government know about you?</li>
+  <li><a href="https://codeberg.org/crimeflare/cloudflare-tor"><strong>The Great Cloudwall</strong></a> - Critique and information on why to avoid Cloudflare, a big company with a huge portition of the internet behind it.</li>
```
Comment (Mikaela) [inline _includes/sections/privacy-resources.html]: I wish to say something about how it's a threat to decentralization, but maybe that is up to the reader to figure out and if they click, they will learn a lot more than my view.

**Pull request merged**